cmd/rootMain.go

```diff
@@ -126,7 +126,7 @@ func rootMain(cmd *cobra.Command, args []string) exitCode {
 
 	if plugin.IsInteractive() {
 		cmdutil.Println(`Welcome to Wash!
-  Wash includes several built-in commands: wexec, find, list, meta, tail.
+  Wash includes several built-in commands: ls, wexec, find, meta, tail.
```

Comment:
I think we should change this to something like
```
Welcome to Wash!
  Type 'help' to view the available commands, 'whistory' to see the commands run,
  and 'whistory <id>' to see any logs associated with a specific command.
Try 'winfo .'!
```

We can then extend `winfo <entry>` (and `docs <entry>`) to look something like
```
Path: /Users/enisinan/Library/Caches/wash/mnt820574140/aws
Name: aws
CName: aws
Actions:
- list (lets you do 'cd <path>', 'ls <path>', and 'tree <path>')
- exec (lets you do 'wexec <path> <command> <args>')
- read (lets you do 'cat <path>', 'grep <regex> <path>', 'less <path>')
...
Attributes: {}
```

This way, the user
* Becomes familiar w/ `winfo` as the main tool for seeing what you can do w/ an entry
* Figures out that they can `ls` the cwd `.`
* Will likely try `winfo` on the plugins themselves, and `cd`/`ls` into them (and repeat this process for the other entries that they can find).

Reply:
Hmm, `Try 'ls'` also reads well and seems natural.

I guess my main concern is exposing the fundamentals of interacting with Wash as early as possible to the user. Everything derives from `winfo` (once we update actions to also list the commands that they work with), which is why I opted for `Try 'winfo'`

Reply:
Main theme here is to try to capture as much of "here are all of the things you can do with Wash" as possible in the main UX so first-time users can immediately dive into the tool. Having people use `winfo` first is a good way of doing that.

I think this new `Actions` output could be verbose (and annoying for people already familiar w/ the actions), so may be worth displaying it via `winfo --verbose`/`winfo -v` ?

Reply:
Also wondering if `winfo` => `winfo .` (maybe `winfo $W`)? Latter makes it clear that `winfo` takes a path.

Reply:
Hmm, I like your idea of a "default, user-friendly" format, and making the structured YAML/JSON output optional (and keep that structured output as it currently is). We could have that "user-friendly" format direct people to "docs" if they'd like more detailed info about the entry, so that way `winfo` also mentions `docs`. I'll think a bit more about how that user-friendly format should look, then file an issue.

Agreed on `Name` and `CName`, though now that the structured output is optional, maybe we can keep as-is?

Re: `winfo .`, agreed. Could probably do that after all the "Make winfo the entry-point to Wash UX" work.